crates/core/src/host/wasmer/wasm_instance_env.rs

```diff
@@ -96,7 +96,7 @@ impl WasmInstanceEnv {
             timing_spans: Default::default(),
             reducer_start,
             call_times: CallTimes::new(),
-            reducer_id: 0,
+            reducer_name: None,
```

Comment:
I don't hugely care: I'm curious if just String is ok, given that accessors to it do map_or("", ...). Like, if there is no semantic difference between an empty name and None, we could just have the one state.